frontend/packages/topology/src/components/page/TopologyView.tsx

```diff
@@ -101,6 +114,7 @@ export const ConnectedTopologyView: React.FC<ComponentProps> = ({
   const [filtersLoaded, setFiltersLoaded] = React.useState<boolean>(false);
   const queryParams = useQueryParams();
   const searchParams = queryParams.get('searchQuery');
+  const fileTypes = fileExtensions.map((ex) => `.${ex}`).toString();
```

Comment:
you can use the FileUploadContext here to get fileExtension if accepting the above-mentioned suggestion.

Reply:
@sahil143  The intention behind passing the extensions as a prop is that I can use it in both DroppableTopology & ConnectedTopology, I do not have to use the context separately for the ConnectedTopology component. Please let me know if I can keep it as is